pkgs/development/compilers/llvm/10/default.nix

```diff
@@ -114,6 +114,22 @@ let
       '' + mkExtraBuildCommands cc;
     };
 
+    cctoolsClangNoLibcxx = wrapCCWith rec {
```

Comment:
There's a lot of duplication here with the `lldClang*` versions. This is complicated enough without having to maintain it over ~5 different clang versions. Is there a better way?

Reply:
Unfortunately, it's not easy to get around. We could add some helper function since lld, cctools, and binutils will all need the same 3 stage process I think (if binutils works at all).